### PR TITLE
Style Layout

### LINKED ISSUE #7: Getting Grid Setup for Layout
**Is your feature request related to a problem? Please describe.**
The page has no styling as of now. 

**Describe the solution you'd like**
Use styled components to create a grid and place the components where they need to go.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Put each component where it is supposed to be

## Purpose
Creates the initial UI

## Approach
Used styled component along with grid

Closes #7

### CODE REVIEW

File: the-toad-tribune/src/layout/Animals.tsx [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+  )
 };
 
 export default Animals;
+
+const AnimalsStyles = styled.div`
+  background-color: white;
+  grid-area: 8/3/10/6;
+  margin-bottom: .5em;
+`
```
Comment (JorgeLVilla) [inline the-toad-tribune/src/layout/Animals.tsx:18]: noiceeee

**Review by JorgeLVilla — APPROVED** [review]
Nice job getting it going!

**Review by JayDampitan — APPROVED** [review]
Thank you grid and thank you Paul for the tips!

File: the-toad-tribune/src/layout/Layout.tsx [hunk 2]
```diff
@@ -34,8 +35,20 @@ const Layout: React.FC<LayoutProps> = ({
...
       {Stonks}
       {Weather}
-    </>
+    </MainLayoutStyles>
   );
 };
 
 export default Layout;
+
+const MainLayoutStyles = styled.div`
```
Comment (Gjhnsn) [inline the-toad-tribune/src/layout/Layout.tsx:44]: styled components looking gooood

**Review by Gjhnsn — APPROVED** [review]
Looking like a great start to your layout!

**Pull request merged**